### ISSUE TITLE
[docs] Add howto create an html email with cozy-notification

### ISSUE THREAD

**Issue by Crash--** [cozy/cozy.github.io#179]
We've a lib called `cozy-notification` used by Banks to create emails and push notifications. We should add an howto use this lib and add a link to it from https://docs.cozy.io/en/cozy-stack/workers/#sendmail-worker

**Comment by ptbrowne** [comment]
Have you seen the readme https://github.com/cozy/cozy-libs/tree/master/packages/cozy-notifications ? It may not be perfect but it's a start.

**Comment by Crash--** [comment]
Yep it's a good start ;) !! I just want to add an how to that covers all this use case: send an html email from my app to my user
